src/main/java/coresearch/cvurl/io/request/RequestWithBodyBuilder.java

```diff
@@ -50,4 +55,29 @@ public RequestWithBodyBuilder body(Object body) {
         header(HttpHeader.CONTENT_TYPE, MIMEType.APPLICATION_JSON);
         return this;
     }
+
+    /**
+     * Sets request body as application/x-www-form-urlencoded. Sets Content-type header
+     * to application/x-www-form-urlencoded.
+     *
+     * @param body request body
+     * @return this builder
+     */
+    public RequestWithBodyBuilder body(Map<Object, Object> body) {
```

Comment:
Form data is not a body actually, users might be confused with this method. I would rather like to see 'payload" or 'formdata", and furthermore - what if i want to have formdata and json body both in single method?